app/models/manageiq/providers/nuage/network_manager/vsd_client/rest.rb

```diff
@@ -32,6 +32,13 @@ def append_headers(key, value)
     @headers[key] = value
   end
 
+  def clear_headers
```

Comment:
May I suggest to use a different method name? `init_headers`, `reset_headers` or something similar. I had a comment asking why are you clearing the headers [here](https://github.com/ManageIQ/manageiq-providers-nuage/pull/97/files#diff-f361099d477f41489f9fa7843a091b63R100). After I saw this, I understand that you are basically resetting them.

Reply:
Good point, thanks. In fact I decided to open a separate PR to fix the vsd client itself so that this one can focus on FloatingIp inventoring only.

https://github.com/ManageIQ/manageiq-providers-nuage/pull/99